base/expr.jl

```diff
@@ -118,6 +118,20 @@ macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
 
+"""
+    @inline
+
+Small functions typically don't even need the @inline annotation,
```

Comment:
Should also make a similar change to the `@inbounds` docstring as well to say what it does.